Do not override flags thereby respect the flags coming from environment,
e.g. we need some optimization level turned on when security flags are enabled
without this change, the build would fail

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Index: ssiapi.1.0.1/configure.ac
===================================================================
--- ssiapi.1.0.1.orig/configure.ac
+++ ssiapi.1.0.1/configure.ac
@@ -15,11 +15,6 @@ AC_SUBST(VERSION)
 AM_INIT_AUTOMAKE(ssi, ${VERSION})
 AM_CONFIG_HEADER(config.h)
 
-dnl Set the language we use
-CPPFLAGS="-g3 -gdwarf-2 -Wall -Werror -D_GNU_SOURCE"
-CFLAGS="-std=gnu99"
-CXXFLAGS="-std=gnu++98"
-
 dnl Automake 1.11 - silent build rules
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
